src/harness/os/windows.c

```diff
@@ -142,15 +170,25 @@ static LONG WINAPI windows_exception_handler(EXCEPTION_POINTERS* ExceptionInfo)
     if (EXCEPTION_STACK_OVERFLOW != ExceptionInfo->ExceptionRecord->ExceptionCode) {
         print_stacktrace(ExceptionInfo->ContextRecord);
     } else {
-        addr2line(windows_program_name, (void*)ExceptionInfo->ContextRecord->Eip);
+#if defined(DETHRACE_CPU_X86) || defined(DETHRACE_CPU_X64)
+        void *addr = (void*)ExceptionInfo->ContextRecord->Eip;
+#elif defined(DETHRACE_CPU_ARM64)
+        void *addr = (void*)ExceptionInfo->ContextRecord->Pc;
+#endif
+        addr2line(windows_program_name, addr);
```

Comment:
Can we rely on having an `addr2line` executable on windows? We shouldn't use `SymGetLineFromAddr64` or something instead?

Reply:
Nope. That's an error.